### PR TITLE
ref(starfish): Extract `SpanMetricsRibbon` component

### PR DESCRIPTION
**Pull request by gggritso**
Continuing the block-u-lation of Starfish view components. This gives me more leverage when deciding whether to specialize components, or use the blocks they're made of.

### CODE REVIEW

File: static/app/views/starfish/views/spanSummaryPage/spanMetricsRibbon.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import {Block, BlockContainer} from 'sentry/views/starfish/views/spanSummaryPage/block';
+import {getSpanOperationDescription} from 'sentry/views/starfish/views/spanSummaryPage/getSpanOperationDescription';
+
+interface Props {
+  spanMetrics: {
+    [SpanMetricsFields.SPAN_OP]: string;
+    [SpanMetricsFields.SPAN_DESCRIPTION]: string;
+    [SpanMetricsFields.SPAN_ACTION]: string;
+    [SpanMetricsFields.SPAN_DOMAIN]: string;
+    [SpanMetricsFields.SPAN_GROUP]: string;
```
Comment (AbhiPrasad) [inline static/app/views/starfish/views/spanSummaryPage/spanMetricsRibbon.tsx:18]: l: Should be optional right?

File: static/app/views/starfish/views/spanSummaryPage/spanMetricsRibbon.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        <Block title={t('Table')}>{spanMetrics?.[SpanMetricsFields.SPAN_DOMAIN]}</Block>
+      )}
+
+      <Block
+        title={getThroughputTitle(op)}
+        description={tct('Throughput of this [opDescription] span per minute', {
+          opDescription,
+        })}
+      >
+        <ThroughputCell rate={spanMetrics?.['spm()']} unit={RateUnits.PER_MINUTE} />
```
Comment (AbhiPrasad) [inline static/app/views/starfish/views/spanSummaryPage/spanMetricsRibbon.tsx:38]: l: should we use the constants for `spm()`, `http_error_count()`,  `time_spent_percentage()` while we here?

**Review by AbhiPrasad — APPROVED** [review]

File: static/app/views/starfish/views/spanSummaryPage/spanMetricsRibbon.tsx (hunk 1 above)
Comment (gggritso) [reply static/app/views/starfish/views/spanSummaryPage/spanMetricsRibbon.tsx]: Done!

File: static/app/views/starfish/views/spanSummaryPage/spanMetricsRibbon.tsx (hunk 2 above)
Comment (gggritso) [reply static/app/views/starfish/views/spanSummaryPage/spanMetricsRibbon.tsx]: Done!

**Pull request merged**